lib/diagnostic.md

```diff
@@ -62,5 +62,7 @@ item elements) to the body DOM node, then hides it.
 Use jQuery to register a click handler for clicks inside `div` items.
 
 <!-- your answer starts here -->
-
+$('div').click(function() {
+    console.log( "You clicked a div!" );
+});
 <!-- your answer ends here -->
```

Comment:
Technically this is right... for now.  This is deprecated however and will not be correct in the near future.  Be sure to check to docs for things like this.